browser/ui/webui/new_tab_page/brave_new_tab_page_handler.cc

```diff
@@ -581,3 +594,24 @@ void BraveNewTabPageHandler::GetBraveBackgrounds(
 
   std::move(callback).Run(std::move(backgrounds));
 }
+
+void BraveNewTabPageHandler::UpdateAudioMuted() {
+  const ContentSetting content_setting =
+      host_content_settings_map_->GetDefaultContentSetting(
+          ContentSettingsType::SOUND);
+
+  const bool should_mute_audio =
+      content_setting != CONTENT_SETTING_ALLOW &&
+      content_setting != CONTENT_SETTING_SESSION_ONLY;
```

Comment:
why are you checking session only? I don't think that even applies to `SOUND`

Reply:
also can we just attach `SoundContentSettingObserver` to this instead?

Reply:
Removed SESSION_ONLY. Thanks

Reply:
Investigating `SoundContentSettingObserver`